Explorer/Assets/DCL/Character/CharacterMotion/Systems/UpdateInputMovementSystem.cs

```diff
@@ -34,7 +34,7 @@ protected override void Update(float t)
 
         [Query]
         [All(typeof(MovementBlockerComponent))]
-        private void ResetInput(ref MovementInputComponent inputToUpdate)
+        private static void ResetInput(ref MovementInputComponent inputToUpdate)
```

Comment:
Does it make a sence to put static on Query?

Reply:
While code-generated Query itself is already static, called method could be created per System instance. However, this particular System exists in only 1 exemplar, so `static` is not very beneficial.. But why not to keep it `static`?

Reply:
If it works with `static` I don't see any reasons why we should remove it